features/networking/egress-ingress.feature

```diff
@@ -269,4 +269,47 @@ Feature: Egress-ingress related networking scenarios
       | curl | --head | www.baidu.com |
     Then the step should succeed
 
+  # @author huirwang@redhat.com
+  # @case_id OCP-13506
+  @admin
+  Scenario: Update different dnsname in same egress network policy
+    Given I have a project
+    Given I have a pod-for-ping in the project
+
+    # Create egressnetworkpolicy to deny www.test.com
+    When I download a file from "https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/dns-egresspolicy/dns-egresspolicy4.json"
```

Comment:
Hard-coded cluster resource name can interfere with other scenarios.

Reply:
@liangxia  Thanks for comments, fix it.